### ISSUE TITLE
Lantern needs to be restarted twice for the new installation to get updated in Ubuntu

### ISSUE THREAD

**Issue by sunshine19090** [getlantern/lantern#3707]
I don't think I can reproduce once I have the latest version. Below are the steps that I followed and found that I had to quit lantern twice before seeing the upgraded version. This might be the case in the previous releases too - am not sure though because I don't remember seeing upgrade for our software in the before releases. It would always say "install". 

Steps:
1. Start running previous version of lantern.
2. Download the installer for 2.1.0 and upgrade it. Lantern will be installed successfully but I didn't see any change in the .lantern folder (no lantern-2.1.0.yaml and settings.yaml file were there)
3. Now go launch lantern again without quitting. You will see that there will be no change in UI. I forgot to see the .lantern folder in this step.
4. Now quit lantern and launch it again. This time, the UI remains the same but the version number is not displayed at the bottom. Also, I didn't get to see the mobile Ad. 
5. So I quit again and launched lantern. This time the .lantern folder contained all the files for 2.1.0 and the UI showed up the mobile Ad with the right version and build number in the footer.

**Comment by myleshorton** [comment]
What OS?

**Comment by myleshorton** [comment]
Oh sorry @sunshine19090 I see Ubuntu now..I'm going to push this to a future version since I don't think it's a blocker for 2.1.0.

**Comment by sunshine19090** [comment]
Yeah sure, sounds good to me. It is  not a blocker. Also I don't think many users would have seen this or they just might have got around like what I did.
